### ISSUE TITLE
A11y_AzureDataStudio_New Deployment:SQL server 2019 Big Data Cluster_Screen Reader: Narrator is not announcing the size of Claim Data in Storage settings.

### ISSUE THREAD

**Issue by Gayathri1501** [microsoft/azuredatastudio#18594]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info & Relationships; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on the Narrator.
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5.  In the Deployment options, select "SQL Server Big Data Cluster" card.
6. Validate the required tools are installed and the "Required Tools" table shows the status as "Installed". Without the required tools the Select button will be disabled.
7. Select the checkbox to accept the license terms and click Select
8. In the Deployment profiles screen, select an appropriate profile and click Next (you can also leave the defaults on)
9. In the Azure settings page, enter the required details and click Next
10. In the Cluster settings enter the required fields (marked with red asterisk) and click Next
11. In the Service settings page, either use the defaults or enter appropriate values and click Next
12. Verify that the size of Claim data in Storage settings is announced by Narrator or not

### Actual Result:
Narrator is not announcing the size of Claim Data in Storage settings.

### Expected Result:
Narrator should announced the size of Claim Data in Storage settings.

### Attachment:
**Narrator Attachment:**
![A11y_AzureDataStudio_New Deployment_ SQL server 2019 Big Data Cluster_Size_Screen Reader](https://user-images.githubusercontent.com/99962579/156163129-ec66fffb-75cf-4ecc-8fbb-2359f937b55f.gif)
**NVDA Attachment:**
**![A11y_AzureDataStudio_New Deployment_SQL server 2019 Big Data Cluster_Size_NVDA](https://user-images.githubusercontent.com/99962579/156163453-2e612513-de73-4a1e-8c0c-a505a2dbd131.gif)**

**Comment by Charles-Gagnon** [comment]
I'm seeing this announced correctly using both NVDA and Narrator so closing this. If you still see an issue please indicate exactly what you expect to see/hear - it isn't clear from your gif what the problem is.

**Comment by Gayathri1501** [comment]
Here, the issue is that GB size if the Claim size of the data is not announcing by the Narrator. The Expected result should be announced by the Narrator. And this bug should be closed by the Trusted tester only. Please confirm us once this bug is fixed.
Verified in the below version,
Version: 1.37.0 (user setup)
Commit: d904740d93d7df76a0ba361f20e4351813b57645
Date: 2022-06-14T01:05:12.352Z
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.25151
![A11y_AzureDataStudio_New Deployment_SQL server 2019 Big Data Cluster_Screen Reader_](https://user-images.githubusercontent.com/99962579/178237555-0c6bda10-29fa-4533-8905-ae557332512d.gif)

**Comment by Charles-Gagnon** [comment]
Thanks for the extra details @Gayathri1501 - the fix has been made and you should see it in the latest version of ADS insiders builds.

**Comment by Charles-Gagnon** [comment]
@Gayathri1501 Please verify the fix and close this issue

**Comment by Gayathri1501** [comment]
Verified in this below version:
Version: 1.38.0 (user setup)
Commit: dff93c4736269ef2a7e8a225737a0d8354bf0246
Date: 2022-07-26T23:56:16.217Z
VS Code: 1.62.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.25163

Issue is not fixed, it is still reproducing in the latest version also. Please find the Attachment.
<img width="960" alt="A11y_AzureDataStudio_New Deployment SQL server 2019 Big Data Cluster_Screen Reader" src="https://user-images.githubusercontent.com/99962579/181444415-91407aaf-e30b-410e-a6c2-234f03fc103e.png">


Please share the environment details where the fixes are available that we regress it and close this bug.

**Comment by Charles-Gagnon** [comment]
@Gayathri1501 Please always use the latest insiders builds when verifying fixes. 

https://github.com/Microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main

Stable builds are only released every other month (approximately) and so fixes made will take a long time to show up there.

**Comment by Gayathri1501** [comment]
Verified this bug in this below version in Insiders version:
Version: 1.39.0-insider (user setup)
Commit: 6be142022080c2c32451afe99f8758cb7c63c52d
Date: 2022-08-01T05:33:09.170Z
VS Code: 1.62.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.25163
<img width="960" alt="A11y_AzureDataStudio_New Deployment SQL server 2019 Big Data Cluster_Screen Reader" src="https://user-images.githubusercontent.com/99962579/182136502-1ecc79ce-b04e-45d3-86e1-1d66d49c7bcb.png">

Issue is Fixed. Hence, closing this bug. Please find the attachment for reference.

**Comment by SachinChavhan317** [comment]
closed this as bug as fixed, hence adding tag #Closed

**Comment by Samhitha66** [comment]
#Closed;
